### ISSUE TITLE
JDK11 build fails during "exploded-image-optimize" target trying to invoke built Java

### ISSUE THREAD

**Issue by andrew-m-leonard** [eclipse/openj9#1478]
During the exploded-image-optimize target of the build where the built jdk Java gets called for the first time, it fails with the exception below.
It looks like src/java.base/share/native/libjava/jni_util.c is expecting an internal variable "fastEncoding" to have been set... but it isn't.
```
Exception in thread "main" java/lang/InternalError: platform encoding not initialized
	at java/io/UnixFileSystem.canonicalize0 (java.base@9/NativeMethod:4294967295)
	at java/io/UnixFileSystem.canonicalize (java.base@9/UnixFileSystem.java:176)
	at java/io/File.getCanonicalPath (java.base@9/File.java:618)
	at java/io/File.getCanonicalFile (java.base@9/File.java:643)
	at jdk/internal/loader/URLClassPath.toFileURL (java.base@9/URLClassPath.java:355)
	at jdk/internal/loader/URLClassPath.<init> (java.base@9/URLClassPath.java:286)
	at jdk/internal/loader/ClassLoaders.<clinit> (java.base@9/ClassLoaders.java:77)
	at java/lang/ClassLoader.initializeClassLoaders (java.base@9/ClassLoader.java:181)
	at java/lang/Thread.initialize (java.base@9/Thread.java:422)
	at java/lang/Thread.<init> (java.base@9/Thread.java:153)
```

**Comment by DanHeidinga** [comment]
See https://github.com/eclipse/openj9/pull/1460

**Comment by pshipton** [comment]
#1460 is merged, but another blocker #1466 is still pending.

**Comment by pshipton** [comment]
#1439 is opened to cover #1466 so I'll close this one.
